src/network/networkd-link.c

```diff
@@ -261,6 +261,20 @@ static int link_enable_ipv6(Link *link) {
         return 0;
 }
 
+static bool link_is_bridge_port(Link *link) {
+
+        assert(link);
+
+        if (link->flags & IFF_LOOPBACK)
+                return false;
+
+        if (link->network)
+                if (link->network->bridge)
+                        return true;
```

Comment:
Again you missing the point. The whole point is if a link is already a bridge port and it's not in the configuration . Have you tested the patch ?